.genx/scripts/update-versions.js

```diff
@@ -20,8 +20,8 @@ const writeJSON = (json, path) => {
 };
 
 const UI = run('npm info @genesislcap/foundation-ui@latest version');
-const GSF = run(`jf rt s "libs-release-client/global/genesis/genesis-distribution/" --sort-by="name;created" --sort-order=desc --limit=1 --exclusions="*-RC*;*-SNAPSHOT*;*maven-metadata*" | grep path | tr -s ' ' | cut -d '/' -f 5`);
-const Auth = run(`jf rt s "libs-release-client/global/genesis/auth-distribution/" --sort-by="name;created" --sort-order=desc --limit=1 --exclusions="*-RC*;*-SNAPSHOT*;*maven-metadata*" | grep path | tr -s ' ' | cut -d '/' -f 5`);
+const GSF = run(`jf rt s "libs-release-client/global/genesis/genesis-distribution/" --sort-by="name;created" --sort-order=desc --limit=1 --exclusions="*-RC*;*-SNAPSHOT*;*maven-metadata*;*test*;*TEST*" | grep path | tr -s ' ' | cut -d '/' -f 5`);
```

Comment:
Just for my understanding, I went through the ticket and it says `*test*,*TEST*` are already included in the exclusions but still don't cover all cases. Just wanted some clarity on that since you haven't added any other exclusions, did you add those there by accident?

Reply:
So if we publish new version of the framework and version name contains any of the characters other than excluded ones specified, the above cli command gives incorrect latest version eg: 7.0.1-TEMP as latest version if we publish version 7.0.1-TEMP. If its not clear, ping me we can discuss on call

Reply:
makes sense, thanks